### PR TITLE
Setup preview branch

### PR DESCRIPTION
**Pull request by funkypenguin**

### CODE REVIEW

File: overrides/main.html [hunk 1]
```diff
@@ -25,3 +25,49 @@
...
+        })
+      }
+    })
+  
+    /* Send page view on document switch (instant loading, custom event) */
+    document.addEventListener("DOMContentSwitch", function() {
+      ga("send", "pageview")
+    })
+  </script>
+  <script async src="https://www.google-analytics.com/analytics.js"></script>
```
Comment (SuperSandro2000) [inline overrides/main.html:72]: So this is really necessary?
Not like I block it at least 3 times.

Comment (funkypenguin) [reply overrides/main.html:72]: I'm looking into alternatives, I agree Google is "icky". Matomo looks good, but their pricing is confusing. I've also experimented with Netlify Analytics, but I found that it didn't provide enough detail.. I'm open to suggestions ;)

Comment (SuperSandro2000) [reply overrides/main.html:72]: I just selfhost matomo.

Comment (funkypenguin) [reply overrides/main.html:72]: I considered that, but the [list of features](https://matomo.org/pricing/) you _don't_ get on self-hosted instances seemed ... overwhelming. Have you felt limited by the self-hosted version?

Comment (SuperSandro2000) [reply overrides/main.html:72]: To be honest I never really digged into the stats to deep, so I didn't notice it.

**Pull request merged**